memestra/nbmemestra.py

```diff
@@ -35,9 +38,9 @@ def nbmemestra_from_nbnode(nb, decorator, filepath):
     return result
 
 
-def nbmemestra(nbfile, decorator, filepath):
+def nbmemestra(nbfile, decorator, filepath, reason_keyword='reason'):
```

Comment:
Some extra thoughts: we could support `reason_keyword=None`, in which case that would catch 


```
@deprectaed("some reason with keywords")
```

And we could also assume that even if a keyword is given, if the decorator call is made without keyword, the reported message should use it.

Does it make sense to you?

Reply:
Yes, that's what I meant here:

> An API design question, what should we do in this case:

I can do that. =)